sig-cloud-provider/annual-report-2023.md

```diff
@@ -12,6 +12,33 @@
    - Governance and leadership changes
 -->
 
+[**Bridget Kromhout and Michael McCune elected as new SIG chairs**](https://groups.google.com/g/kubernetes-sig-cloud-provider/c/BAh7gGZhfvo/m/5eJtVdoIDwAJ)
+
+Previous chairs Andrew Sy Kim and Nick Turner stepped down during 2023, with Andrew staying on as a technical lead and Nick becoming an emeritus lead. Many thanks for their leadership and guidance over the years.
```

Comment:
(I realize that we're writing this "As of Jan 1, 2024..." so Andrew is still tech lead in the report, but we do need the README to be current, not point-in-time.)

Reply:
right, i was going to move Nick now because that is an old change that we need to reflect, and then wait for the process on moving Andrew.

Reply:
if i understand correctly, we'll update in #7815 ?